datahub-actions/src/datahub_actions/plugin/source/kafka/kafka_event_source.py

```diff
@@ -201,16 +217,69 @@ def close(self) -> None:
             self.running = False
             self.consumer.close()
 
-    def ack(self, event: EventEnvelope) -> None:
-        self.consumer.commit(
+    def _with_retry(self, func: Callable, *args, **kwargs) -> Any:  # type: ignore
```

Comment:
nitpick: i'd prefer to see this in a sibling utils.py file

Reply:
move to utils.py